webrtc/main.go

```diff
@@ -85,11 +87,15 @@ func main() {
 
 	inboundRTPPacket := make([]byte, 4096) // UDP MTU
 
+	var once sync.Once
+
 	// Read RTP packets forever and send them to the WebRTC Client
 	for {
 
 		n, _, err := listener.ReadFrom(inboundRTPPacket)
 
+		once.Do(func() { fmt.Print("houston we have a packet") })
```

Comment:
Missed a log message

Reply:
I want to keep this one in since its useful to know when we receive the first RTP packet. I plan on overhauling the logging here next though and will make it nicer :)